lab-data/yocto-beagleplay/bootlin-lab-data/nunchuk/linux/0002-arm64-dts-ti-k3-am625-beagleplay.dts-add-description.patch

```diff
@@ -11,42 +11,32 @@ Content-Transfer-Encoding: 8bit
 mikrobus needs to be disabled
 
 Signed-off-by: Jérémie Dautheribes <jeremie.dautheribes@bootlin.com>
+Signed-off-by: Antonin Godard <antonin.godard@bootlin.com>
 ---
- arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts | 14 +++++++++++++-
- 1 file changed, 13 insertions(+), 1 deletion(-)
+ arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts | 9 ++++++++-
+ 1 file changed, 8 insertions(+), 1 deletion(-)
 
 diff --git a/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts b/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts
-index d779fd110d4a..bac06eca433c 100644
+index 89cb616e1d1f..1b9c7b91562f 100644
 --- a/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts
 +++ b/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts
-@@ -282,6 +282,7 @@ cpsw3g_phy1: ethernet-phy@1 {
- 	};
- 
- 	mikrobus0: linux-mikrobus {
-+		status = "disabled";
- 		compatible = "linux,mikrobus";
- 		pinctrl-names = "default", "pwm_default", "pwm_gpio",
- 				"uart_default", "uart_gpio", "i2c_default",
-@@ -1055,7 +1056,18 @@ it66121hdmitx_out: endpoint {
- 
+@@ -828,8 +828,15 @@ it66121_out: endpoint {
  &main_i2c3 {
- 	status = "okay";
+ 	pinctrl-names = "default";
+ 	pinctrl-0 = <&mikrobus_i2c_pins_default>;
 -	clock-frequency = <400000>;
 +	clock-frequency = <100000>;
```

Comment:
This is of course not a problem fir the training, but it is strange. Is there a valid reason for decreasing the bus speed?

Reply:
Cf. Greg: the nunchuk we use in the lab does not support more than 100kHz

Reply:
Sensible reason!